tests/ha/single_pod_restart/test_server_pod_restart_api.py

```diff
@@ -131,17 +135,38 @@ def setup_method(self):
             self.num_replica = int(resp[1])
         else:
             self.num_replica = 1
+        LOGGER.info("Create IAM user")
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        self.s3_clean.update(users)
+        access_key = list(users.values())[0]["accesskey"]
+        secret_key = list(users.values())[0]["secretkey"]
+        self.s3_test_obj = S3TestLib(access_key=access_key, secret_key=secret_key,
+                                     endpoint_url=S3_CFG["s3_url"])
+        self.s3_ver = S3VersioningTestLib(access_key=access_key, secret_key=secret_key,
+                                          endpoint_url=S3_CFG["s3_url"], region=S3_CFG["region"])
+        LOGGER.info("Created IAM user")
+        if self.setup_type == "VM":
+            self.f_size = str(HA_CFG["5gb_mpu_data"]["file_size_512M"]) + "M"
+        else:
+            self.f_size = str(HA_CFG["5gb_mpu_data"]["file_size"]) + "M"
```

Comment:
"MB"

Reply:
Default is "M" in main function and its working
def create_file(fpath: str, count: int, dev: str = "/dev/zero", b_size: str = "1M") -> tuple: